### PR TITLE
[Sluggable] Separate slug transliterator

### PR DESCRIPTION
**Pull request by sergeyfedotov**
This feature allows to use custom slug transliteration class.

### CODE REVIEW

File: src/Model/Sluggable/SluggableMethods.php [hunk 1]
```diff
@@ -68,7 +68,7 @@ public function getSlug()
      * @param $values
      * @return mixed|string
      */
-    private function generateSlugValue($values)
+    private function generateSlugValue($values, TransliteratorInterface $transliterator)
```
Comment (docteurklein) [inline src/Model/Sluggable/SluggableMethods.php]: I'm not sure i like the idea of injecting the transliterator during the method call.

**Comment by akovalyov** [comment]
Any activity/holywars on it?

**Comment by MAXakaWIZARD** [comment]
@akovalyov You could just override `generateSlug` method and use any transliterator you want, I did it in such way before my PR #162 was accepted and merged.

**Comment by TomasVotruba** [comment]
Is this feature still needed? Could you rebase if so?

**Comment by TomasVotruba** [comment]
As the service is static hard-coded in object and serves only as default implementaion,
overriding it is the way to go:
https://github.com/KnpLabs/DoctrineBehaviors/pull/224#issuecomment-125553897

Closing as resolved

**Pull request closed without merging**